sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_manual_validation_info_async.py

```diff
@@ -35,11 +35,15 @@ class GetValidationInfoSampleAsync(object):
     async def get_manual_validation_info_from_recognize_custom_forms(self):
```

Comment:
I'm wondering if we don't want to call this something more general than manual validation info.  It's showing how to get bounding boxes, yes?  Those might also be used for drawing UI as part of an application.

Reply:
switched to sample_get_detailed_bounding_box_information